src/js/components/DataSearch/DataSearch.js

```diff
@@ -42,17 +67,14 @@ export const DataSearch = ({ drop, id: idProp, responsive, ...rest }) => {
       name="_search"
       icon={<Search />}
       type="search"
+      onChange={onChange}
       {...rest}
     />
   );
 
   if (!inDataForm)
     // likely in Toolbar
-    content = (
-      <DataForm footer={false} updateOn="change">
-        {content}
-      </DataForm>
-    );
+    content = <Box>{content}</Box>;
```

Comment:
So we don't lose the reasoning in the future, can we add a comment mentioning that presence of this Box ensures the right default width.

Reply:
Certainly